libbeat/template/field.go

```diff
@@ -1,238 +0,0 @@
-package template
-
-import (
-	"fmt"
-	"strings"
-
-	"github.com/elastic/beats/libbeat/common"
-)
-
-var (
-	defaultScalingFactor = 1000
-)
-
-type Field struct {
-	Name           string      `config:"name"`
-	Type           string      `config:"type"`
-	Description    string      `config:"description"`
-	Format         string      `config:"format"`
-	ScalingFactor  int         `config:"scaling_factor"`
-	Fields         Fields      `config:"fields"`
-	MultiFields    Fields      `config:"multi_fields"`
-	ObjectType     string      `config:"object_type"`
-	Enabled        *bool       `config:"enabled"`
-	Analyzer       string      `config:"analyzer"`
-	SearchAnalyzer string      `config:"search_analyzer"`
-	Norms          bool        `config:"norms"`
-	Dynamic        dynamicType `config:"dynamic"`
-	Index          *bool       `config:"index"`
-	DocValues      *bool       `config:"doc_values"`
-
-	path      string
-	esVersion common.Version
-}
-
-// This includes all entries without special handling for different versions.
-// Currently this is:
-// long, geo_point, date, short, byte, float, double, boolean
-func (f *Field) other() common.MapStr {
```

Comment:
This logic and the other methods regarding transformation moved to `template.Processor`.